docs/lists/data/lists.json

```diff
@@ -37,25 +37,6 @@
       "unsub" : "http://groups.google.com/group/ack-users",
       "url" : null
    },
-   "activeperl" : {
-      "archive" : "https://code.activestate.com/lists/activeperl/",
-      "category" : "Win32 Ports",
-      "comments" : "",
-      "date_last_confirmed_active" : "2015-01-27 12:45:00",
-      "defunct" : "0",
```

Comment:
instead of deleting the list, mark it as "defunct" for archival purposes.

Reply:
Hello, thank you for the hint, I noticed that some lists were in the list but not in the perl.org nor my local dev website but haven't connected to this "variable".